apps/omg_status/lib/omg_status/alert/alarm_handler.ex

```diff
@@ -39,6 +39,7 @@ defmodule OMG.Status.Alert.AlarmHandler do
   # -----------------------------------------------------------------
   def init(_args) do
     table_setup()
+
```

Comment:
Actually not your code, but I'm trying to avoid `for` as long as it realizes std `map` or `each` operation.
Can you replace below 👇  Thx

Reply:
done